code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm

```diff
@@ -214,19 +214,48 @@
 	name = "12g flechette box"
 	result = /obj/item/ammo_box/shotgun/flechette
 	reqs = list(/obj/item/stack/crafting/metalparts = 1,
-	/obj/item/stack/sheet/prewar = 2,
+	/obj/item/stack/sheet/prewar = 50,//Yes, really.
```

Comment:
I suggest replacing the pre-war metal requirement with either 2 or 3 polymers (alien alloy), or a pre-war steel ingot